tools/cldr-apps/pom.xml

```diff
@@ -0,0 +1,309 @@
+      <version>2.3.1</version>
+      <scope>provided</scope>
+    </dependency>
+
+
+    <!-- mail / rss -->
+    <dependency>
+      <groupId>rome</groupId>
+      <artifactId>rome</artifactId>
+      <version>1.0</version>
+    </dependency>
+
+    <dependency>
+      <groupId>javax.mail</groupId>
+      <artifactId>mail</artifactId>
+      <version>1.5.0-b01</version>
+    </dependency>
+
+
+    <dependency>
+      <groupId>com.sun.activation</groupId>
+      <artifactId>javax.activation</artifactId>
+      <version>1.2.0</version>
+    </dependency>
+
+<!-- https://mvnrepository.com/artifact/org.json/json -->
+<dependency>
+    <groupId>org.json</groupId>
+    <artifactId>json</artifactId>
+    <version>20190722</version>
+</dependency>
+
+    <!-- scm -->
+    <dependency>
+      <groupId>org.tmatesoft.svnkit</groupId>
+      <artifactId>svnkit</artifactId>
+      <version>1.7.4-v1</version>
+    </dependency>
+
+    <!-- codec/util -->
+    <dependency>
+      <groupId>commons-codec</groupId>
+      <artifactId>commons-codec</artifactId>
+      <version>1.9</version>
+    </dependency>
+    <dependency>
+      <groupId>commons-fileupload</groupId>
+      <artifactId>commons-fileupload</artifactId>
+      <version>1.3.1</version>
+    </dependency>
+    <dependency>
+      <groupId>commons-io</groupId>
+      <artifactId>commons-io</artifactId>
+      <version>2.4</version>
+    </dependency>
+<!-- https://mvnrepository.com/artifact/com.google.collections/google-collections -->
+<!-- <dependency>
+  <groupId>com.google.collections</groupId>
+  <artifactId>google-collections</artifactId>
+  <version>1.0</version>
+</dependency> -->
+    
+    <!-- db -->
+    <dependency>
+      <groupId>org.apache.derby</groupId>
+      <artifactId>derby</artifactId>
+      <version>10.10.1.1</version>
+      <!-- <scope>test</scope> -->
+    </dependency>
+
+    <dependency>
+    <groupId>mysql</groupId>
+    <artifactId>mysql-connector-java</artifactId>
+    <version>8.0.21</version>
+    </dependency>
+
+<!-- https://mvnrepository.com/artifact/com.google.guava/guava -->
+<dependency>
+    <groupId>com.google.guava</groupId>
+    <artifactId>guava</artifactId>
+    <version>28.1-jre</version>
+</dependency>
+<!-- Note: failureaccess comes in automatically. -->
+
+<!-- https://mvnrepository.com/artifact/com.google.code.gson/gson -->
+<dependency>
+    <groupId>com.google.code.gson</groupId>
+    <artifactId>gson</artifactId>
+    <version>2.8.5</version>
+</dependency>
+
+<dependency>
+    <groupId>com.google.myanmartools</groupId>
+    <artifactId>myanmar-tools</artifactId>
+    <version>1.1.1</version>
+</dependency>
+
+<dependency>
+    <groupId>xerces</groupId>
+    <artifactId>xercesImpl</artifactId>
+    <version>2.12.0</version>
+</dependency>
+
+    
+    
+  </dependencies>
+
+  <build>
+    <finalName>cldr-apps</finalName>
+    <sourceDirectory>src</sourceDirectory> <!-- TODO: HACK! until we can pivot src dirs -->
```

Comment:
Can you reference a CLDR ticket number in the TODO ?

Reply:
It's this same one, CLDR-11585. To solve this I need to rearrange the source files which breaks everyone. That's why I want To do this in stages.